cli/cmd/deploy.go

```diff
@@ -63,12 +63,16 @@ func buildDeployServiceCommand(srv string) *cobra.Command {
 		Short: `Deploys a ` + srv + ` service`,
 		Long:  `Deploys a ` + srv + ` service, adding it to a running profile, identified by its name`,
 		Run: func(cmd *cobra.Command, args []string) {
-			serviceManager := compose.NewServiceManager()
```

Comment:
Rather than calling `deploy.NewServiceManager` we could initialize it like we do in the fleet tests that way you just use `deployer.Add` instead and it'll do the right thing no matter the provider (docker or k8s)

Reply:
If we do that we will be introducing a bug in the CLI side, as we have coupled the Bootstrap method with the Fleet profile. Maybe we can fix that in a separate PR.

Reply:
Well, indeed the bug is there if we decide to migrate the metricbeat test suite 😄